app/assets/javascripts/discourse/controllers/bookmark.js

```diff
@@ -71,9 +111,44 @@ export default Controller.extend(ModalFunctionality, {
     }
   },
 
+  bindKeyboardShortcuts() {
+    if (!this.mouseTrap) {
+      this.mouseTrap = Mousetrap(
+        document.querySelector("#discourse-modal.bookmark-with-reminder")
+      );
+    }
+
+    GLOBAL_SHORTCUTS_TO_PAUSE.forEach(combo => Mousetrap.unbind(combo));
```

Comment:
Wondering, does it work:

```
 GLOBAL_SHORTCUTS_TO_PAUSE.forEach(Mousetrap.unbind)
```

?

Reply:
No it doesn't, though I would expect it to :/